src/components/editor/EditorActiveLinePanel.tsx

```diff
@@ -0,0 +1,44 @@
+import { EditorView, Extension, Panel, showPanel } from '@uiw/react-codemirror'
+
+const activeLinePanelTheme = EditorView.baseTheme({
+    '.cm-active-panel': {
+        padding: '4px 6px',
+        fontSize: '13px',
+        backgroundColor: '#292c34',
+        color: '#A4ABB7',
+    },
+    '.cm-panels.cm-panels-bottom': {
+        borderTop: '1px solid #16181D !important',
```

Comment:
Seemed like `!important` was needed here due to the way these styles were added, but possibly should be injected elsewhere to avoid this?